app/src/main/res/layout/fragment_summary_product.xml

```diff
@@ -377,6 +392,14 @@
                 android:layout_marginTop="@dimen/spacing_tiny"
                 android:text="@string/take_more_pictures" />
 
+            <Button
+                android:id="@+id/add_nutrition_prompt"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_horizontal"
+                android:layout_marginTop="4dp"
+                android:text="Add nutrients prompt"/>
```

Comment:
You should change the text on the button to be more meaningful to the user.

Reply:
Type nutrition facts to get the Nutri-Score